packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -1,12 +1,7 @@
 <template>
   <weblet-layout ref="layout">
     <template #title>
-      Deployment List
-      {{
-        $props.projectName
-          ? "(" + (tabs.find(tab => tab.value === $props.projectName)?.title ?? $props.projectName) + ")"
-          : ""
-      }}
+      {{ title }}
```

Comment:
```suggestion
      {{ title || "Deployment List" }}
```
Make title optional to allow to use component as before (might be needed later)